modules/zivid/camera.py

```diff
@@ -1,210 +1,92 @@
 
+        Raises:
+            TypeError: If argument is neither a Settings or a Settings2D
         """
-        if settings_collection is not None:
+        if isinstance(settings, zivid.Settings):
             return Frame(
+                self.__impl.capture(_settings_converter.to_internal_settings(settings))
+            )
+        if isinstance(settings, Settings2D):
+            return Frame2D(
                 self.__impl.capture(
-                    [
-                        _settings_converter.to_internal_settings(settings)
-                        for settings in settings_collection
-                    ]
+                    _settings_2d_converter.to_internal_settings2_d(settings)
                 )
             )
-        return Frame(self.__impl.capture())
-
-    def capture_2d(self, settings_2d):
-        """Capture a single 2D frame.
+        raise TypeError("Unsupported settings type: {}".format(type(settings)))
 
-        Note that the provided settings will only apply to this current 2D capture, and not future 3D captures.
-
-        Args:
-            settings_2d: Settings to use for the capture
+    @property
+    def info(self):
+        """Get information about camera model, serial number etc.
 
         Returns:
-            A frame containing a 2D image and metadata.
-
+            A CameraInfo instance
         """
-        return Frame2D(
-            self.__impl.capture_2d(
-                _settings_2d_converter.to_internal_settings_2d(settings_2d)
-            )
-        )
+        return _camera_info_converter.to_camera_info(self.__impl.info)
 
     @property
     def state(self):
         """Get the current camera state.
 
         Returns:
-            The current camera state
-
+            A CameraState instance
         """
         return _camera_state_converter.to_camera_state(self.__impl.state)
 
-    @property
-    def settings(self):
-        """Get the current camera settings.
-
-        Returns:
-            Current settings
-
-        """
-        return _settings_converter.to_settings(self.__impl.settings)
-
-    @settings.setter
-    def settings(self, settings):
-        """Update the camera settings.
-
-        Args:
-            settings: New settings for the camera
-
-        """
-        self.__impl.settings = _settings_converter.to_internal_settings(  # pylint: disable=protected-access
-            settings
-        )
-
-    def connect(self, settings=None):
-        """Connect to the camera.
-
-        Args:
-            settings: New settings for the camera
-
-        """
-        if settings is None:
-            self.__impl.connect()
-        else:
-            self.__impl.connect(
-                _settings_converter.to_internal_settings(  # pylint: disable=protected-access
-                    settings
-                )
-            )
+    def connect(self):
+        """Connect to the camera."""
+        self.__impl.connect()
```

Comment:
I don't see why the user would ever need that, as it's basically just a method that modifies the object. 
`newcamera = camera.connect()` is really weird since both variables will refer to the same camera.

On the other hand, in the C++ API camera.connect() does return a reference to the same camera. Not sure what to do here..

Reply:
Did we fix `disconnect`? Is `disconnect` returning a camera?
Not sure if this makes sense either, but I heard an argument, at least for C++, that some users would like to chain these calls. Not sure if this applies for zivid-python as well.

Reply:
In the C++ API, connect() resturns a Camera reference and disconnect() returns void. I guess there is an argument for wanting to do chaining like: `camera.connect().capture()`. We already made support for chaining in the case of PointCloud.transform() and PointCloud.downsample().

Given these arguments, I think we should make connect() in zivid-python also return a camera reference. I'll do that now.

Reply:
I added a return value to camera.connect() now. Also added a unit-test to ensure that chaining works.